lib/Onnxifi/onnxifiGlow.cpp

```diff
@@ -57,67 +70,50 @@ GLOW_ONNXIFI_LIBRARY_FUNCTION_WRAPPER(onnxGetBackendIDs)(
   const size_t numBackendsCapacity = *numBackends;
 
   using namespace glow::runtime;
+  using namespace glow::onnxifi;
   const bool withCPU = DeviceManager::numDevices("CPU") > 0;
   const bool withHabana = DeviceManager::numDevices("Habana") > 0;
 
   // Only return quantization backend if GLOW_DUMP_PROFILE.
   if (getenv("GLOW_DUMP_PROFILE")) {
-    *numBackends = 2;
+    *numBackends = 1;
 
     // In case backendIDs is nullptr or does not have enough capacity just
     // return the total number of supported backends.
     if (numBackendsCapacity < *numBackends || !backendIDs) {
       return ONNXIFI_STATUS_FALLBACK;
     }
 
-    auto *quantizationBackendOnnx = manager.createBackend(
-        "Interpreter", /*useOnnx*/ true, /*forQuantization*/ true);
     auto *quantizationBackendC2 = manager.createBackend(
-        "Interpreter", /*useOnnx*/ false, /*forQuantization*/ true);
-
-    backendIDs[0] = quantizationBackendOnnx;
-    backendIDs[1] = quantizationBackendC2;
-  } else if (withCPU || withHabana) {
-    *numBackends = 4;
+        GlowOnnxifiBackend.empty() ? "Interpreter" : GlowOnnxifiBackend,
+        /*useOnnx*/ false, /*forQuantization*/ true);
```

Comment:
So are we never going to use onnx for quantization profiling via onnxifi?

Reply:
yup, no more use of onnx for inference or profiling.